.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: rust
 rust:
-  - 1.20.0
```

Comment:
Then can we make this check for 1.28?

Reply:
My intent is to merge this alongside https://github.com/servo/rust-smallvec/pull/162, if I provided a different version, this would cause a merge conflict.